### PR TITLE
MUI. Material `5.12.3` & Lab `5.0.0-alpha.129` [generated]

### PR DESCRIPTION
**Pull request by aerialist7**

### CODE REVIEW

File: kotlin-mui/src/jsMain/generated/mui/base/useList.types.kt [hunk 1]
```diff
@@ -174,3 +174,15 @@ external interface ListItemState {
      */
     var selected: Boolean
 }
+
+external interface UseListReturnValue {
+    var contextValue: dynamic
+
+    var dispatch: (action: Any /* CustomAction | ListAction<ItemValue> */) -> Unit
+
+    var getRootProps: react.Props /* <TOther extends EventHandlers = {}>(otherHandlers?: TOther) => UseListRootSlotProps<TOther> */
```
Comment (turansky) [inline kotlin-mui/src/jsMain/generated/mui/base/useList.types.kt:183]: Looks like function

File: kotlin-mui/src/jsMain/generated/mui/base/useSlider.types.kt [hunk 2]
```diff
@@ -194,6 +194,11 @@ external interface UseSliderReturnValue {
      */
     var range: Boolean
 
+    /**
+     * Ref to the root slot's DOM node.
+     */
+    var rootRef: react.RefCallback<web.html.HTMLElement>?
```
Comment (turansky) [inline kotlin-mui/src/jsMain/generated/mui/base/useSlider.types.kt:200]: `import web.html.HTML`?

File: kotlin-mui/src/jsMain/generated/mui/base/useTabsList.types.kt [hunk 3]
```diff
@@ -6,16 +6,28 @@ external interface UseTabsListParameters {
...
      */
-
+    var rootRef: react.Ref<web.html.HTMLElement>
 }
 
 external interface UseTabsListReturnValue {
+    /**
+     * The value to be passed to the TabListProvider above all the tabs.
+     */
+    var contextValue: dynamic
```
Comment (turansky) [inline kotlin-mui/src/jsMain/generated/mui/base/useTabsList.types.kt:16]: `Any`?

File: kotlin-mui/src/jsMain/generated/mui/base/useTabsList.types.kt [hunk 4]
```diff
@@ -6,16 +6,28 @@ external interface UseTabsListParameters {
...
      * Allows to programmatically control the tabs list.
      */
     var dispatch: (action: Any /* ListAction<string | number> */) -> Unit
 
+    /**
+     * Resolver for the root slot's props.
+     * @param externalProps props for the root slot
+     * @returns props that should be spread on the root slot
+     */
+    var getRootProps: react.Props /* <TOther extends Record<string, any> = {}>(externalProps?: TOther) => UseTabsListRootSlotProps<TOther> */
```
Comment (turansky) [inline kotlin-mui/src/jsMain/generated/mui/base/useTabsList.types.kt:29]: Looks like function

File: kotlin-mui/src/jsMain/generated/mui/material/TextField.kt [hunk 5]
```diff
@@ -126,6 +126,11 @@ external interface BaseTextFieldProps :
 
     var onFocus: react.dom.events.FocusEventHandler<web.html.HTMLElement>?
 
+    /**
+     * @ignore
```
Comment (turansky) [inline kotlin-mui/src/jsMain/generated/mui/material/TextField.kt:130]: What does it mean?

File: kotlin-mui/src/jsMain/generated/mui/base/useTabsList.types.kt (hunk 3 above)
Comment (aerialist7) [reply kotlin-mui/src/jsMain/generated/mui/base/useTabsList.types.kt:16]: Only the order changed

File: kotlin-mui/src/jsMain/generated/mui/base/useTabsList.types.kt (hunk 4 above)
Comment (aerialist7) [reply kotlin-mui/src/jsMain/generated/mui/base/useTabsList.types.kt:29]: Only the order changed

File: kotlin-mui/src/jsMain/generated/mui/material/TextField.kt (hunk 5 above)
Comment (aerialist7) [reply kotlin-mui/src/jsMain/generated/mui/material/TextField.kt:130]: From mui docs

File: kotlin-mui/src/jsMain/generated/mui/base/useSlider.types.kt (hunk 2 above)
Comment (aerialist7) [reply kotlin-mui/src/jsMain/generated/mui/base/useSlider.types.kt:200]: Will become `web.dom.Element` in the next update

File: kotlin-mui/src/jsMain/generated/mui/base/useList.types.kt (hunk 1 above)
Comment (aerialist7) [reply kotlin-mui/src/jsMain/generated/mui/base/useList.types.kt:183]: It should be a separate change for all functions like this one. Next time

**Pull request merged**